.gitignore

```diff
@@ -88,3 +88,6 @@ Example/fastlane/test_output/
 # https://github.com/johnno1962/injectionforxcode
 
 iOSInjectionProject/
+
+# jazzy
+Example/docs
```

Comment:
I wonder that whether I can read this document right on GitHub? That looks better, right? And a link on readme to lead me to this 😁

Reply:
Already put it in branch `docs`, you can check this link: https://nimblehq.github.io/local-authentication-ios/ 😁